.npmignore

```diff
@@ -0,0 +1,7 @@
+# ignore everything
```

Comment:
FYI: npm nimmt als Fallback die .gitignore als npm ignore file her. Falls die Dateien sich also eh gleichen sollten, ist das vielleicht interessant zu wissen :)
Hier ist jetzt keine Änderung nötig. Wollte ich nur als Info da lassen. Siehe: https://npm.github.io/publishing-pkgs-docs/publishing/the-npmignore-file.html

Reply:
Ja, die npm ignore ist bewusst da, weil ich mehr ausschließen will als in der igtignore :)